src/common/sidebar-injector.js

```diff
@@ -303,7 +303,7 @@ function SidebarInjector(chromeTabs, dependencies) {
     if (!isSupportedURL(tab.url)) {
       return Promise.resolve();
     }
-    return injectScript(tab.id, '/public/destroy.js');
+    return injectScript(tab.id, '/lib/destroy.js');
```

Comment:
`destroy.js` is actually not Chrome-specific at all, so it can go in `src/common`